src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1720,8 +1723,7 @@ public void updateTriggerConfigURL() {
                 triggerInformationAction.setErrorMessage("Dynamic trigger configuration needs "
                         + "a specific configured server");
             } else {
-                List<GerritProject> fetchedProjects = GerritDynamicUrlProcessor.fetch(triggerConfigURL, serverName);
-                dynamicGerritProjects = fetchedProjects;
+                dynamicGerritProjects = GerritDynamicUrlProcessor.fetch(triggerConfigURL, serverName);
```

Comment:
Why this change?
If there is an error now you will be stuck with not triggering on anything instead of at least triggering on what was parsed in the last fetch.

Reply:
if `fetch` will raise a exception - assignment will not happen. Do not see any difference - just use one variable less.